apps/studio/lib/constants/telemetry.ts

```diff
@@ -372,3 +371,72 @@ export interface SqlEditorResultCopyMarkdownClickedEvent {
 export interface SqlEditorResultCopyJsonClickedEvent {
   action: TelemetryActions.SQL_EDITOR_RESULT_COPY_JSON_CLICKED
 }
+
+/**
+ * User submitted a prompt to the assistant sidebar.
+ *
+ * @group Events
+ * @source studio
+ */
+export interface AssistantPromptSubmittedEvent {
+  action: TelemetryActions.ASSISTANT_PROMPT_SUBMITTED
+}
+
+/**
+ * User submitted a debug request to the assistant sidebar or prompt submitted has Help me to debug.
+ *
+ * @group Events
+ * @source studio
+ */
+export interface AssistantDebugSubmittedEvent {
+  action: TelemetryActions.ASSISTANT_DEBUG_SUBMITTED
+}
+
+/**
+ * User ran a suggestion provided by the assistant sidebar.
+ *
+ * @group Events
+ * @source studio
+ */
+export interface AssistantSuggestionRanEvent {
+  action: TelemetryActions.ASSISTANT_SUGGESTION_RAN
```

Comment:
<img width="846" alt="Screenshot 2024-12-31 16 58 40@2x" src="https://github.com/user-attachments/assets/6ece8a20-ee39-4205-a838-2aa7208ecd76" />

Some suggestion to keep it in line with the [event tracking standarization](https://www.notion.so/supabase/Event-tracking-standardization-1195004b775f80f98ee3fa9e70cf4d05).

```suggestion
  action: TelemetryActions.ASSISTANT_SUGGESTION_EXECUTION_CLICKED
```

Reply:
Because the button name is run query, will change it to RUN_QUERY_CLICKED, together with ACCEPT_CLICKED and REJECT_CLICKED